cmake/FairMQConfig.cmake.in

```diff
@@ -14,9 +14,9 @@ set(@PROJECT_NAME@_GIT_VERSION @PROJECT_GIT_VERSION@)
 set(@PROJECT_NAME@_GIT_DATE @PROJECT_GIT_DATE@)
 
 set_and_check(@PROJECT_NAME@_ROOT @PACKAGE_CMAKE_INSTALL_PREFIX@)
-set_and_check(@PROJECT_NAME@_BINDIR @PACKAGE_CMAKE_INSTALL_PREFIX@/@PROJECT_INSTALL_BINDIR@)
-set_and_check(@PROJECT_NAME@_INCDIR @PACKAGE_CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_INCLUDEDIR@)
-set_and_check(@PROJECT_NAME@_LIBDIR @PACKAGE_CMAKE_INSTALL_PREFIX@/@PROJECT_INSTALL_LIBDIR@)
+set_and_check(@PROJECT_NAME@_BINARY_DIR @PACKAGE_CMAKE_INSTALL_PREFIX@/@PROJECT_INSTALL_BINDIR@)
+set_and_check(@PROJECT_NAME@_INCLUDE_DIRS @PACKAGE_CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_INCLUDEDIR@)
+set_and_check(@PROJECT_NAME@_LIBRARIES @PACKAGE_CMAKE_INSTALL_PREFIX@/@PROJECT_INSTALL_LIBDIR@)
```

Comment:
According to the manpage you linked, this one should be `_LIBRARY_DIRS`. In addition, it is probably good to also set `_LIBRARIES`, but if I understand correctly, it should contain the absolute path to libFairMQ.

Reply:
I am a bit confused about the difference between _LIBRARY_DIRS and _LIBRARIES aswell. If i understand correctly the only difference is that one is a relative path and the other is an absolute path? 
I can't find any other project that uses them both.

Reply:
Also thank you for your response! 
I will indeed process your bulletpoints

Reply:
> If i understand correctly the only difference is that one is a relative path and the other is an absolute path?

I think, `_LIBRARIES` is used as a list of names to pass as `-l` arguments to the linker. According to the CMake manual it should contain the list of absolute paths to the library files, one has to link against.

Also note, that we are exporting a target named `FairMQ::FairMQ` that has all the information above set as target properties. So, instead of relying on these variables, in modern CMake (3.0+) we pass around such paths and filenames as target properties. But I have no objections against setting the old style variables as well. So, thx again for your contribution.